packages/@aws-cdk/aws-workspaces/package.json

```diff
@@ -20,7 +20,11 @@
         "signAssembly": true,
         "assemblyOriginatorKeyFile": "../../key.snk"
       },
-      "sphinx": {}
+      "sphinx": {},
+      "awsdocs": {
+        "title": "WorkSpaces",
```

Comment:
Open question - should we prefix with Amazon or AWS respectively? (I'd assume this would be the "first occurrence" on the page, but I'd gladly agree this feels redundant).